client/blocks/comment-detail/style.scss

```diff
@@ -586,6 +586,18 @@ a.comment-detail__author-more-element {
 		font-size: 14px;
 	}
 
+	.comment-detail__edit-jetpack-update-notice {
+		color: $alert-red;
+		margin-left: 8px;
+		margin-right: 8px;
+
+		a {
```

Comment:
If we give the link a classname, we can break this out of the nesting. We only expect one link in the notice, correct?